### PR TITLE
[TS-184] clean dependency: web

### PR DESCRIPTION
**Pull request by jenchik**

### CODE REVIEW

**Review by kpeo — DISMISSED** [review]

File: web/web.go [hunk 1]
```diff
@@ -20,20 +18,38 @@ const (
 	defaultBind = ":8080"
 )
 
+type recoverer interface {
```
Comment (im-kulikov) [inline web/web.go:22]: not exporting really?

Comment (jenchik) [reply web/web.go:22]: for now it's no sense

**Review by kpeo — DISMISSED** [review]

File: web/web.go [hunk 2]
```diff
@@ -80,3 +116,11 @@ func StartServer(e *echo.Echo, bind string) error {
 
 	return nil
 }
+
+// AddTrailingSlash returns a root level (before router) middleware which adds a
+// trailing slash to the request `URL#Path`.
+//
+// Usage `Engine#Pre(AddTrailingSlash())`
+func AddTrailingSlash() MiddlewareFunc {
```
Comment (im-kulikov) [inline web/web.go:133]: please, add another one method - `RemoveTrailingSlash`

File: web/web.go [hunk 3]
```diff
@@ -20,20 +19,57 @@ const (
...
+	Context = echo.Context
+
+	// HandlerFunc from echo.HandlerFunc (for shadowing)
+	HandlerFunc = echo.HandlerFunc
+
+	// MiddlewareFunc from echo.MiddlewareFunc (for shadowing)
+	MiddlewareFunc = echo.MiddlewareFunc
+
+	// Engine from echo.Echo (for shadowing)
+	Engine = echo.Echo
```
Comment (im-kulikov) [inline web/web.go:67]: mb we can extract interface?

Comment (jenchik) [reply web/web.go:67]: for now it's no sense

File: web/web.go (hunk 2 above)
Comment (jenchik) [reply web/web.go:133]: fixed

File: web/web.go (hunk 3 above)
Comment (im-kulikov) [reply web/web.go:67]: why not?

Comment (jenchik) [reply web/web.go:67]: not need

**Review by im-kulikov — APPROVED** [review]
echo.Echo we can improve later.. 
post-2500-0-31507700-1480324422

**Review by kpeo — DISMISSED** [review]

File: web/web.go (hunk 3 above)
Comment (im-kulikov) [reply web/web.go:67]: maybe you wanna to talk about this

**Review by im-kulikov — DISMISSED** [review]

**Review by kpeo — APPROVED** [review]

**Review by pcherednichenko — APPROVED** [review]

File: web/web.go [hunk 4]
```diff
@@ -66,37 +68,44 @@ type (
...
 	e.HideBanner = true
-	e.Logger = opts.Logger
 
 	if opts.Validator != nil {
 		e.Validator = opts.Validator
 	}
 
-	e.HTTPErrorHandler = opts.Error.Capture
-	e.Use(opts.Error.Recover())
+	if opts.Error != nil {
```
Comment (im-kulikov) [inline web/web.go:105]: 👍🏻

**Review by im-kulikov — APPROVED** [review]

**Pull request merged**